java-vtl-script/src/test/java/no/ssb/vtl/script/VTLScriptEngineTest.java

```diff
@@ -1132,7 +1132,11 @@ public void testOuterJoinWithMultipleIds() throws ScriptException {
         Dataset result = (Dataset) bindings.get("result");
         assertThat(result.getDataStructure().getRoles()).containsOnly(
                 entry("id1", Role.IDENTIFIER),
+                //entry("a_id1", Role.IDENTIFIER),
```

Comment:
Shouldn't those commented lines be removed?

Reply:
Fixed